main.tf

```diff
@@ -196,6 +196,9 @@ resource "google_compute_instance_template" "default" {
   # In order to update an Instance Template, Terraform will destroy the existing resource and create a replacement
   lifecycle {
     create_before_destroy = true
+    ignore_changes = [
+      disk[0].source_image
```

Comment:
I'm not sure if this will work when the `var.disk_source_image` is set actually.. I would need to test this..